### PR TITLE
note on dynamic roles caveat

### PR DESCRIPTION
**Pull request by DaanHoogland**

### CODE REVIEW

**Comment by DaanHoogland** [comment]
@blueorangutan docbuild

File: source/adminguide/accounts.rst [hunk 1]
```diff
@@ -135,6 +135,17 @@ allows CloudStack root admins to create new roles with customized permissions.
 The allow/deny rules can be configured dynamically during runtime without
 restarting the management server(s).
 
+.. Note:: Any user given the custom roles that include permission to
+          create and/or update accounts will have the ability to
+          assign new custom roles to themsevles or other users,
+          irrspective of the privialges given in those roles. This
```
Comment (sureshanaparti) [inline source/adminguide/accounts.rst:141]:
```suggestion
          irrespective of the privileges given in those roles. This
```

File: source/adminguide/accounts.rst [hunk 2]
```diff
@@ -135,6 +135,17 @@ allows CloudStack root admins to create new roles with customized permissions.
...
+          could allow such a user to escalate their own privalges to
```
Comment (sureshanaparti) [inline source/adminguide/accounts.rst:142]:
```suggestion
          could allow such a user to escalate their own privileges to
```

File: source/adminguide/accounts.rst [hunk 3]
```diff
@@ -135,6 +135,17 @@ allows CloudStack root admins to create new roles with customized permissions.
...
+          include any API they might not have had before. Thereofre,
```
Comment (sureshanaparti) [inline source/adminguide/accounts.rst:143]:
```suggestion
          include any API they might not have had before. Therefore,
```

File: source/adminguide/accounts.rst [hunk 4]
```diff
@@ -135,6 +135,17 @@ allows CloudStack root admins to create new roles with customized permissions.
...
+          the dynamic roles should be carefully designed and the
+          `createAccount` and `updateAccount` priviledges should only
```
Comment (sureshanaparti) [inline source/adminguide/accounts.rst:145]:
```suggestion
          `createAccount` and `updateAccount` privileges should only
```

File: source/adminguide/accounts.rst [hunk 5]
```diff
@@ -138,11 +138,11 @@ restarting the management server(s).
...
-          could allow such a user to escalate their own privalges to
-          include any API they might not have had before. Thereofre,
+          irrespective of the privileges given in those roles. This
+          could allow such a user to escalate their own privileges to
+          include any API they might not have had before. Therefore,
           the dynamic roles should be carefully designed and the
-          `createAccount` and `updateAccount` priviledges should only
+          `createAccount` and `updateAccount` privileges should only
           be given to users who you are content to have this level of
           privilage
```
Comment (sureshanaparti) [inline source/adminguide/accounts.rst:147]:
```suggestion
          privilege
```

one more

**Comment by sureshanaparti** [comment]
@blueorangutan docbuild

**Review by GabrielBrascher — APPROVED** [review]
LGTM.
Clear, concise, and in the right place.
Thanks, @DaanHoogland!

**Comment by sureshanaparti** [comment]
@blueorangutan docbuild

**Comment by sureshanaparti** [comment]
Verified doc details here http://qa.cloudstack.cloud/docs/WIP-PROOFING/pr/260/adminguide/accounts.html#using-dynamic-roles

**Review by sureshanaparti — APPROVED** [review]

**Review by rohityadavcloud — COMMENTED** [review]
LGTM

**Pull request merged**